frontend/src/components/shareProject/ProjectTypeSelector.tsx

```diff
@@ -0,0 +1,108 @@
+import {
+  Card,
+  Grid,
+  makeStyles,
+  Radio,
+  RadioGroup,
+  Typography,
+} from "@material-ui/core";
+import React from "react";
+import { getImageUrl } from "../../../public/lib/imageOperations";
+
+const useStyles = makeStyles((theme) => ({
+  reset: {
+    margin: 0,
+    padding: 0,
+    listStyleType: "none",
+    width: "100%",
+  },
+  box: {
+    borderRadius: 3,
+    margin: theme.spacing(1),
+    boxShadow: "3px 3px 8px #E0E0E0",
+    cursor: "pointer",
+    display: "flex",
+    color: theme.palette.secondary.main,
+    alignItems: "center",
+    position: "relative",
+  },
+  boxActive: {
+    backgroundColor: "#d7f1ef",
+    border: `1px solid ${theme.palette.primary.main}`,
+  },
+  textBox: {
+    width: 225,
+    marginTop: theme.spacing(2),
+    marginBottom: theme.spacing(2),
+    marginRight: theme.spacing(4),
+  },
+  headline: {
+    fontWeight: "bold",
+  },
+  helpText: {
+    fontSize: 14,
+  },
+  imageContainer: {
+    width: 60,
+    margin: theme.spacing(2),
+    display: "flex",
+    alignItems: "stretch",
+    height: "100%",
+  },
+  image: {
+    height: "100%",
+    width: "100%",
+  },
+  radioButton: {
+    position: "absolute",
+    right: 0,
+    top: 0,
+  },
+}));
+
+export default function ProjectTypeSelector({ className, value, types, onChange }) {
```

Comment:
Nit: Can you move the default function at the end of the file? Its just easier to read and follow.